composer.json

```diff
@@ -9,8 +9,8 @@
     ],
     "require": {
         "php": ">=7.1.0",
-        "silverstripe/framework": "^4.5",
-        "silverstripe/admin": "^1.5",
+        "silverstripe/framework": "^4.6",
```

Comment:
@maxime-rainville just wondering if I should revert the ^4.6 back to ^4.5 - I did this without thinking.  I don't believe there's anything in the version of cwp-core that explicitly requires 4.6 of framework, or 1.6 of admin

Reply:
Maybe leave it like that. cwp-core is kind of an almost recipe.